x-pack/docs/en/security/authentication/service-accounts.asciidoc

```diff
@@ -34,13 +37,20 @@ Service accounts are not included in the response of the
 Service accounts have a
 <<security-api-get-service-accounts-path-params,unique principal>> that takes
 the format of `<namespace>/<service>`, where the `namespace` is a top-level
-grouping of service accounts, and `service` is the name of the service.
+grouping of service accounts, and `service` is the name of the service and
+must be unique within its namespace.
 
-Currently, only one service account is available:
+Service accounts are predefined in code. Currently, only one service account is available:
 
 `elastic/fleet-server`:: The service account used by the {fleet} server to
 communicate with {es}.
 
+While not recommended, it is possible to create a regular user with the same
+username as a service account principal. There will be no security complications
+because the {es} views them as distinct, separate identities. In addition, a service
+account can only be authenticated with service tokens which is not applicable for
+regular users.
+
```

Comment:
@ywangd, this information contradicts the note that follows. If we don't recommend that users create a user with the same username as a service account principal, I think we should actively discourage this behavior. I think we can remove this text and amend the following note to read:

> IMPORTANT: Do not attempt to use service accounts for authenticating individual users. Service accounts can only be authenticated with service tokens, which are not applicable to regular users.

Reply:
I agree that we should discourage it. I was just trying to be comprehensive. But it does add to confusion as a side effect. I am happy for you to amend it as you see fit.